3rd_party/dependency_report.cmake

```diff
@@ -15,46 +15,45 @@
 # that is required for the stack-wide list.
 #
 # Usage:
-# dependency_report.sh --csv <output_file>
+# cmake -D OUTPUT_FILE=<output_file> -P dependency_report.cmake
```

Comment:
Please keep the existing `dependency_report.sh` in parallel to this new script until we can move release manager over to the new script.